backend/memory/memory.go

```diff
@@ -37,15 +38,15 @@ type TimeSeries struct {
 }
 
 type Tenant struct {
-	ts map[string]*TimeSeries
+	ts                 map[string]*TimeSeries
+	alerts 			   alerts.Alerts
```

Comment:
Please do not.

You can implement **all** you need for alerts using the backend interface !!
this way we will not need to re-invent the weel for each new db backend.

use: the methods available here:
https://github.com/MohawkTSDB/mohawk/blob/master/backend/backend.go#L58

Reply:
Cool okay I understand.

Reply:
p.s.

you can add new methods to the interface if you think you are missing something